mocha/mocha.opts

```diff
@@ -0,0 +1,4 @@
+--recursive
+--webpack-config webpack.config.js
+--require mocha/setup.js
+--timeout 10s
```

Comment:
I don't really like increasing timeouts. Tests should *never* have any reason take more than 2s, which I think you said is the default? If timeouts are happening, something is possibly wrong with the test. Which test is it?